gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
We could use something else instead of 1 too. (like 3, 4, 5 ...). It's not about collapsing. The idea is about a known value to exit with, when it reaches the `||` condition.

> An exit status of 1 doesn't mean the command failed, it means input wasn't sorted.

That is gnu sort oddity and as you shared, it will be applicable "if invoked with -c or -C".